scripts/cipublish

```diff
@@ -62,6 +62,16 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                 "s3://${RF_ARTIFACTS_BUCKET}/lambda-functions/package-${GIT_COMMIT}.zip"
             rm "${DIR}/../app-lambda/package.zip"
 
+            echo "Uploading API specs"
+            # At the most recent version prefixed
+            aws s3 sync \
+                "${DIR}/../docs/swagger/" \
+                "s3://${RF_DOCS_BUCKET}/spec/$(git describe --abbrev=0 --tags)/"
+            # At the root
+            aws s3 sync \
+                "${DIR}/../docs/swagger/" \
+                "s3://${RF_DOCS_BUCKET}/spec/"
```

Comment:
Doesn't seem to be any logging output from these commands, which is fine, though "Uploading API specs", then silence, is a bit out-of-character for our normally comically verbose aws cli logging

Reply:
There will be some, but only when things are changing.